torch/utils/cpp_extension.py

```diff
@@ -964,6 +964,16 @@ def CppExtension(name, sources, *args, **kwargs):
     constructor. Full list arguments can be found at
     https://setuptools.pypa.io/en/latest/userguide/ext_modules.html#extension-api-reference
 
+    Note!
```

Comment:
This _is_ mostly copy/pasta with the CUDAExtension description...I _could_ move it to deduplicate but then we'd have to do our setting __doc__ trick which is less readable in the code....